Thanks @lieck . The patch lgtm!

But you still need to fix the golang-lint:

```
unit/type/json/json_test.go:60: File is not `gofmt`-ed with `-s` (gofmt)
	    require.NoError(t, rdb.Do(ctx, "SET", "a", `1`).Err())
        require.Error(t, rdb.Do(ctx, "JSON.ARRAPPEND", "a", "$", `1`).Err())
        require.NoError(t, rdb.Do(ctx, "DEL", "a").Err())
$ /home/runner/go/bin/golangci-lint run -v ./...
Traceback (most recent call last):
```

You can just run `go fmt` to format the file.

--- a/tests/gocase/unit/type/json/json_test.go
+++ b/tests/gocase/unit/type/json/json_test.go
@@ -57,9 +57,9 @@ func TestJson(t *testing.T) {
 	})
 
 	t.Run("JSON.ARRAPPEND basics", func(t *testing.T) {
-	    require.NoError(t, rdb.Do(ctx, "SET", "a", `1`).Err())
-        require.Error(t, rdb.Do(ctx, "JSON.ARRAPPEND", "a", "$", `1`).Err())
-        require.NoError(t, rdb.Do(ctx, "DEL", "a").Err())
+		require.NoError(t, rdb.Do(ctx, "SET", "a", `1`).Err())
+		require.Error(t, rdb.Do(ctx, "JSON.ARRAPPEND", "a", "$", `1`).Err())
+		require.NoError(t, rdb.Do(ctx, "DEL", "a").Err())
 
 		require.NoError(t, rdb.Do(ctx, "JSON.SET", "a", "$", ` {"x":1, "y": {"x":1} } `).Err())
 		require.Equal(t, []interface{}{}, rdb.Do(ctx, "JSON.ARRAPPEND", "a", "$..k", `1`).Val())

